### ISSUE TITLE
Feature Request: File Zipping

### ISSUE THREAD

**Issue by talaniz** [loics2/sorta#9]
I know you haven't touched this in a while, but it just occurred to me while I was doing a repetitive task that file zipping would be a really good feature to have. Maybe zipping and deleting the original.

**Comment by loics2** [comment]
Yeah I was quite busy, and I'll be until this summer, but I begun to implement a more generic architecture allowing to use "plugins". With this architecture, you'll be able to add a zip action.

Keep an eye here, I'll try to push the new code as soon as possible
